lib/string_decoder.js

```diff
@@ -14,6 +16,8 @@ function normalizeEncoding(enc) {
   return nenc || enc;
 }
 
+const simpleWrite = {};
```

Comment:
Using a prototypeless object would be *better*, but I'm not sure that having a lookup object like this is *best*. Have you ran benchmarks?